config/.env

```diff
@@ -1 +1 @@
-NOTIFICATION_URL="https://ddehnr4ewobxc.cloudfront.net/dynNotifications.json"
\ No newline at end of file
+NOTIFICATION_URL="https://ddehnr4ewobxc.cloudfront.net/dynNotifications.json"
```

Comment:
Should we leave this empty for production after your change @filipeotero ?

Reply:
Yes, otherwise it will get the notifications by itself.

Reply:
Commented out